spec/classes/plugin/nginx_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe 'letsencrypt::plugin::nginx' do
+  on_supported_os.each do |os, facts|
+    context "on #{os} based operating systems" do
```

Comment:
We generally keep it short
```suggestion
    context "on #{os}" do
```